functions/colab.js

```diff
@@ -42,6 +42,22 @@ exports.handler = async function (event, _) {
     .map((v) => v[0].toUpperCase() + v.slice(1))
     .join(' ')
   // notebook cell structure
+  let common_commands = [
+    `!wget ${zipRes}\n`,
+    `!unzip ${template}.zip\n`,
+    '!pip install -r requirements.txt'
+  ]
+
+  let specific_commands = []
```

Comment:
Nice idea, what do you think about empty cell if there is no specific code to run ?
<img width="1114" alt="image" src="https://github.com/pytorch-ignite/code-generator/assets/2459423/aa57153a-b092-43c8-9e3b-2ff47efdeead">

Reply:
Yeah, I was thinking of keeping a separate dict variable so we could add an if condition for specific templates... So the empty cell wouldn't be in templates not required.

Reply:
Have made the changes, how do they look? I'm worried I'm using too many variables...

Reply:
Yes, the code now looks a bit ugly, let's find a middle ground between previous version and this one